src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User validUserBob;
+    private static User validUserKyle;
+    private static User validUserGreg;
+    private static User userWithNotValidLogin;
+    private static User userWithNotValidAge;
+    private static User userWithNotValidPassword;
```

Comment:
no need to create that many variables
not valid users can be easily converted to local variables